lambda/Makefile

```diff
@@ -21,14 +21,26 @@ checks: clean
 	isort --atomic -yq
 	black -q .
 	flake8 . --max-line-length=91
-	#mypy --pretty .
+	mypy --warn-unused-config \
+             --disallow-any-generics \
+             --disallow-subclassing-any \
+             --disallow-untyped-calls \
+             --disallow-untyped-defs \
+             --disallow-incomplete-defs \
+             --check-untyped-defs \
+             --no-implicit-optional \
+             --warn-redundant-casts \
+             --warn-unused-ignores \
+             --warn-return-any \
+             --no-implicit-reexport \
+             --strict-equality .
```

Comment:
Wow, this looks cool! Does this make mypy more strict? I'm delighted!

Reply:
This is basically all the options from `--strict` except `--disallow-untyped-decorators` as that was breaking.

```  --strict                  Strict mode; enables the following flags: --warn-unused-configs, --disallow-any-generics, --disallow-subclassing-any, --disallow-untyped-calls, --disallow-
                            untyped-defs, --disallow-incomplete-defs, --check-untyped-defs, --disallow-untyped-decorators, --no-implicit-optional, --warn-redundant-casts, --warn-unused-
                            ignores, --warn-return-any, --no-implicit-reexport, --strict-equality```

Reply:
oh oh OH. Also good.